### PR TITLE
Remove preview features doc and update feature flagging.

### PR DESCRIPTION
**Pull request by iAmWillShepherd**
Reading through the docs, noticed that we don't call out the features under preview. 

@desktop/core feels on screenshots or gifs to illustrate each feature?

### CODE REVIEW

**Comment by shiftkey** [comment]
@iAmWillShepherd this document is kind of sad, because we're just now mentioning features that have been on the beta channel for a while.

I think we should move the technical details about `GITHUB_DESKTOP_PREVIEW_FEATURES` into `docs/technical/feature-flagging.md` (it already gets a mention) and figure out a better way to share what stuff is currently on the beta channel that end users will be more likely to notice.

**Comment by iAmWillShepherd** [comment]
Sounds good. And yea, I just noticed we had nothing there, so I figured I'd add it while no one was looking 😆

File: docs/technical/feature-flagging.md [hunk 1]
```diff
@@ -49,6 +49,9 @@ feature once things are stabilized.
...
 
-To opt-in for testing preview features, set the
-`GITHUB_DESKTOP_PREVIEW_FEATURES` environment variable to any value and launch
-the Desktop app.
+Set the `GITHUB_DESKTOP_PREVIEW_FEATURES` environment variable to any value to opt-in for preview featurs.
+
+Don't have that environment variable? 
+No worries, simply create it. (here's a [handy guide](https://www.schrodinger.com/kb/1842) for doing that on most major OSs).
+
+Note, if you would like to opt-out of preview feautures, you will need to remove this variable from your environment.
```
Comment (j-f1) [inline docs/technical/feature-flagging.md]: Do you have to restart Desktop each time you change the setting?

Comment (iAmWillShepherd) [reply docs/technical/feature-flagging.md]: Good point. We should make that clear.

**Comment by iAmWillShepherd** [comment]
🔢

File: docs/contributing/preview-features.md [hunk 2]
```diff
@@ -1,14 +1,3 @@
...
-feedback from users on these features.
-
-To opt-in for testing preview features, set the
-`GITHUB_DESKTOP_PREVIEW_FEATURES` environment variable to any value and launch
-the Desktop app.
-
-## Preview Features
-
-Coming Soon™
+We are hard at work building the latest features, we will have something to show soon...
```
Comment (shiftkey) [inline docs/contributing/preview-features.md]: Let's just remove this whole file (and open an issue to track this idea properly) :fire::fire::fire:

**Comment by iAmWillShepherd** [comment]
🔢

**Pull request merged**